sdk/storage/storage-queue/swagger/README.md

```diff
@@ -107,6 +107,10 @@ directive:
     where: $.parameters.VisibilityTimeoutRequired
     transform: >
       $["x-ms-client-name"] = "visibilityTimeout";
+  - from: swagger-document
+    where: $.parameters.VisibilityTimeoutForEnqueue
```

Comment:
I am a little surprised that without the swagger transformation, the codegen will pick the documentation from another parameter.  @joheredi can you have a look to see whether it's expected behavior in autorest?

Reply:
Actually we also updated the swagger. We used the same paramter for the two parmeters in enqueue and dequeue in the swagger, while their description is different. So I think the codegen is working as expected.
https://github.com/Azure/azure-rest-api-specs/pull/11459

Reply:
It looks like the Swagger already sets the ` x-ms-client-name` to "visibilityTimeout". Do we need to have it here as well?

Reply:
The `"x-ms-client-name": "visibilityTimeout"` in swagger is removed in https://github.com/Azure/azure-rest-api-specs/pull/11461.
Other languages use a different casing so we have to do it in our own README.md.